packages/cloud_firestore/cloud_firestore/lib/src/firestore.dart

```diff
@@ -315,6 +315,8 @@ class FirebaseFirestore extends FirebasePluginPlatform {
       persistenceEnabled: settings.persistenceEnabled,
       host: settings.host,
       cacheSizeBytes: settings.cacheSizeBytes,
+      webExperimentalForceLongPolling: settings.webExperimentalForceLongPolling,
+      webExperimentalAutoDetectLongPolling: settings.webExperimentalAutoDetectLongPolling,
```

Comment:
I'm not seeing timeoutSeconds exposed in the user facing lib. I think it should be hidden being an objet like in the original web SDK. [experimentalLongPollingOptions](https://firebase.google.com/docs/reference/js/firestore_.firestoresettings.md#firestoresettingsexperimentallongpollingoptions)